### PR TITLE
soyjs: i18n msg support

### PR DESCRIPTION
**Pull request by mhupman**

### CODE REVIEW

**Comment by mhupman** [comment]
@robfig ping!

**Comment by robfig** [comment]
Sorry, I'm still not getting notified from new issues :\.  Any chance of getting a test or two in here for this?

File: soyjs/exec.go [hunk 1]
```diff
@@ -502,42 +503,96 @@ func (s *state) visitSwitch(node *ast.SwitchNode) {
...
+	if msg == nil {
+		s.visitMsgNode(node, node, nil)
+		return
+	}
+
+	s.visitMsgNode(node, node, msg.Parts)
+}
+
+func (s *state) visitMsgNode(n ast.ParentNode, root *ast.MsgNode, parts []soymsg.Part) {
+	for i, child := range n.Children() {
```
Comment (robfig) [inline soyjs/exec.go]: might be worth adding a sanity check that parts is the same length as children?  (or is it nil if no bundle is provided?)

Comment (mhupman) [inline soyjs/exec.go]: Sure - would a panic with some detail be appropriate in that case?

**Comment by mhupman** [comment]
I'll work on adding some tests.

**Comment by mhupman** [comment]
Closing, I think this is horribly broken after attempting some tests.  Will resubmit.

**Pull request closed without merging**